index.html

```diff
@@ -19,37 +19,39 @@
     <h1 class="text-center mt-2 text-xl text-gray-200 font-semibold">
       <a
         href="https://www.instagram.com/frederiquelamazone/"
-        class="rounded-lg bg hover:text-blue-800 hover:bg-white"
+        class="rounded-lg bg hover:text-blue-800 text-shadow"
       >
         @frederiquelamazone
       </a>
     </h1>
 
     <!-- CONTENU -->
     <div class="flex justify-between">
-      <div class="flex align-center justify-center ml-12">
-        <img src="images/arrow.png" class="w-24" />
+      <div
+        class="hidden sm:hidden md:flex lg:flex xl:flex align-center justify-center ml-12 pb-8 w-auto lg:w-20 xl:w-20"
```

Comment:
`hidden` and `sm:hidden` are redundant.
`md:flex`, `lg:flex` and `xl:flex` are redundant.
`lg:w-20` and `xl:w-20` are redundant.

Read this: https://tailwindcss.com/docs/responsive-design.

Reply:
done